kpatch-build/create-diff-object.c

```diff
@@ -321,6 +321,7 @@ static int is_special_static(struct symbol *sym)
 		"__FUNCTION__.",
 		"_rs.",
 		"CSWTCH.",
+		".L.str",
```

Comment:
> Is there a reason these variables are treated as "special" statics? If so it should be documented in the commit message.

I forgot to circle back to this before posting. So the reason for that was that L.str symbols can be referenced from __dyndebug section which is never correlated which in turn made kpatch_correlate_static_local_variables() fail. I think it will be better addressed by skipping correlation of symbols in these sections.